dom5.js

```diff
@@ -455,9 +457,63 @@ function newElement(tagName, namespace) {
   };
 }
 
+function newDocumentFragment() {
+  return {
+    nodeName: '#document-fragment',
```

Comment:
Parse5's `parser.parseFromFragment` returns basically this, but with `quirksMode: false`. Probably not important, but just noting it here.